src/components/Earn.jsx

```diff
@@ -31,10 +38,10 @@ const YieldgenReport = ({ lines, maxAmountOfRows = 25 }) => {
 
   return (
     <div className="mt-2 mb-3">
-      {empty && <rb.Alert variant="info">{t('earn.alert_empty_report')}</rb.Alert>}
+      {empty && <Alert variant="info">{t('earn.alert_empty_report')}</Alert>}
```

Comment:
As far as I can see our own `Alert` component does not support `children`, but a `message` property. Does this work on your end?

Reply:
Nice catch! This is supposed to be a react-bootstrap component, I accidentally imported the local one. Fixed!